packages/playground/remote/src/lib/boot-playground-remote.ts

```diff
@@ -230,6 +230,20 @@ function setupPostMessageRelay(
 
 		window.parent.postMessage(event.data, '*');
 	});
+
+	window.addEventListener('message', (event) => {
+		if (event.source !== window.parent) {
+			return;
+		}
+
+		if (typeof event.data !== 'object' || event.data.type !== 'relay') {
+			return;
+		}
+
+		console.log("passing message to iframe's window", event);
```

Comment:
I'd remove this console.log from here. If you want some confirmation the relay is actually set up, the setupPostMessageRelay function itself could log something like "Setting up a postMessage relay".

Reply:
Can confirm this was testing.